lib/id3c/cli/command/identifier.py

```diff
@@ -46,6 +46,11 @@ def identifier():
     metavar = "<file.pdf>",
     type = click.File("wb"))
 
+@click.option("--small",
```

Comment:
Could we make this more generic, for example, make the option "--layout" to choose which layout to use? This use case has us going with the small layout, but we may have other scenarios that have us use multiple layouts for a single collection set. If --layout is not specified use the default.

Reply:
Interesting. Were you thinking `--layout` would be the Cryo layout specification, something like `--layout LCRY-2380`? In that approach, we'd also need to specify options like `--no-barcode-text` or `--no-barcode-name`. I'm not sure how useful those latter options would be.

Reply:
A hybrid approach could be to use `--layout small` (or `compact`? `minified`?) as the option to toggle both the label size and whether text is included. `--layout` could be expanded to support other layouts in the future.

Reply:
I like the hybrid approach of declaring a `--layout` type with a common name (like small/compact/minified etc.). I also see how creating a `--layout` option could more easily be expanded to support other layouts in the future, but I'm concerned about the "YAGNI" philosophy @tsibley often raises. How likely are we to have to support future layouts?

Reply:
I would weigh two factors: 
- YAGNI and the likeliness of future alternate layouts: I don't know of specific plans (but it reasonably could happen)
- the difficulty to change it later: in this case, refactoring it later seems doable, and we are the users of this CLI which gives us a little more flexibility.

Based on the above understanding, `--small` is ok with me for now, with the assumption we would likely refactor it if we do need another alternate layout later. (A comment to that effect, either in the code or in the commit message, may be a useful reminder of that intention)

Since he raised the point initially, I want to give @jccraft-uw a chance to weigh back in here as well.

Reply:
Thanks for taking a look, @kschwabefry . In the interest of getting this out sooner than later, I agree with going with `--small`. If we need additional layouts in the future we can refactor the code then.

Reply:
What about an option where we offer `--layout` with a default value of `default`, and then this PR implements a `--layout` value of `small`?

And I'll implement Kristen's other suggestion to raise a NotImplementedError if `small` or some other non-supported layout is specified but there's nothing implemented for that class.

Reply:
`--layout` sounds really good. We're specifying which layout we want. For now, only CLIA would have a layout other than `default`, `small`.
This is what I had in mind--specify the layout that you want when you're minting.

Reply:
Sounds like a good resolution. Thanks all.

Reply:
Thanks both for your feedback. Please take a look at the latest commit. The only thing I'm not sure of is if we want to maintain the layout choices in two places: one in the Click option in the `identifier` command, and the other in the set of implemented layouts for each barcode layout in `labelmaker`.

Reply:
I would not add the possible options in the Click option because it would take work to keep that up to date if we add layouts in labelmaker.